lib/tasks/migrate/20240404154458_add_uuid_to_medias.rake

```diff
@@ -1,18 +1,27 @@
 namespace :check do
   namespace :migrate do
+    def claim_uuid_for_duplicate_quote
+      puts "Collect Claim uuid for duplicate quotes"
+      claim_uuid = {}
+      Claim.select('lower(quote) as quote, MIN(medias.id) as first')
+      .joins("INNER JOIN project_medias pm ON pm.media_id = medias.id")
+      .group('lower(quote)').having('COUNT(medias.id) > 1')
+      .each do |raw|
+        claim_uuid[raw['quote']] = raw['first'].to_s
```

Comment:
@melsawy , I think it will be faster and safer (in terms of memory consumption, etc.) to use a hash of the `quote` as the key, for example: `Digest::MD5.hex(raw['quote'])`